font/ttf.ksy

```diff
@@ -11,7 +11,9 @@ meta:
 doc: |
   A TrueType font file contains data, in table format, that comprises
   an outline font.
-doc-ref: https://www.microsoft.com/typography/tt/ttf_spec/ttch02.doc
+doc-ref:
+  - https://www.microsoft.com/typography/tt/ttf_spec/ttch02.doc
```

Comment:
This link no longer works (it is redirected to https://docs.microsoft.com/typography), so it would nice to delete or replace it with something else (when you're already updating the links).

The original `ttch02.doc` is still available via the Wayback Machine (https://web.archive.org/web/20160410081432/https://www.microsoft.com/typography/tt/ttf_spec/ttch02.doc), but the question is whether this resource brings any additional value.

Reply:
The extra `doc-ref` I added has a more complete description of the current format, which might, or might not, be pure TrueType. Most "TrueType" fonts these days are actually OpenType fonts (but converted to TrueType format, with TrueType magic, and so on). It has become a bit of a mix it seems.

Reply:
I also added a reference to Apple's TrueType documentation. Many of the fonts that I see that are TrueType are a combination of what I see in Apple's documentation and Microsoft's documentation.

Because TrueType and OpenType are nearly identical (except the first four bytes) I think that it might be worth making this a more generic font description.